packages/core/src/components/widgets/index.ts

```diff
@@ -38,3 +40,5 @@ export default {
   CheckboxesWidget,
   SubmitButton,
 };
+
+export default widgets;
```

Comment:
Also, once you've made this change, I believe you can update the `getDefaultRegistry()` to remove the type cast there